src/index.html

```diff
@@ -10,13 +10,51 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header"
+      id="top"
+    >
+      <a>
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
```

Comment:
The `<a>` tag wrapping the logo image should have an `href` attribute to make it a functional link. Consider adding a URL to the `href` attribute to direct users to the homepage or another relevant page.